client/src/worldgen.rs

```diff
@@ -124,15 +125,40 @@ pub fn voxels(graph: &DualGraph, node: NodeId, chunk: Vertex) -> VoxelData {
                 let center = voxel_center(coords);
                 let cube_coords = center * 0.5;
 
-                let (voxel_mat, elevation_boost) = match trilerp(&enviros.rainfalls, cube_coords) {
-                    r if r > 2.0 => (Material::Dirt, 1.0),
-                    r if r < 1.0 => (Material::Stone, -1.0),
-                    _ => (Material::Sand, -0.5),
-                };
+                let elev = trilerp(&enviros.max_elevations, cube_coords);
+                let rain = trilerp(&enviros.rainfalls, cube_coords);
+                let temp = trilerp(&enviros.temperatures, cube_coords);
+                let slope = trilerp(&enviros.slopeiness, cube_coords);
+
+                let mut voxel_mat = Material::Dirt;
+                let max_e = elev;
```

Comment:
I don't think we need two different variables for this. Just replace use of max_e with elev?

Reply:
I disagree. Max_e will eventually be modified by other factors.

Reply:
Max_e is an ouput, elev is an input.